src/main/java/eu/eutampieri/catacombs/ui/DungeonGame.java

```diff
@@ -95,4 +95,13 @@ public void endGame() {
         setState(this.transition.transState("Dead", this, endGame));
     }
 
+    /**
+     * This method start the winning transition transition.
+     */
+
+    public void winGame() {
+        final EndGameState endGame = new EndGameState(this);
+        setState(this.transition.transState("WIN!!", this, endGame));
```

Comment:
A better message would be "Game won" or you've won the game!

Reply:
done